### DISCUSSION TITLE
5.1.2 Released

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4296] [General]
Blog announcement here: https://www.oqtane.org/blog/!/89/oqtane-5-1-2-released

9 contributors to the 5.1.2 release is the most ever for an Oqtane release! Thank you so much for your contributions!

**Comment by vnetonline** [comment]
My hosting provider has installed 8.0.6 will 5.1.2 work on it ?

**Reply by sbwalker** [reply]
it should work fine.... I did not know that Microsoft was going to release .NET 8.0.6 yesterday

**Comment by vnetonline** [comment]
The automatic update didn't work. I had to manually unzip the upgrade package after stopping the website 

https://LearnOqtane.com now running 5.1.2 on hosting provider [SmarterASP.NET](https://www.SmarterASP.NET/index?r=vnetonline) who upgraded my server to ASP.NET Core runtime 8.0.6

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/9e662694-b4bc-4fde-a772-4661fabbc8f7)
